app/src/main/java/com/example/poste/activities/EditPostActivity.java

```diff
@@ -2,52 +2,32 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
-import android.app.AlertDialog;
-import android.app.ProgressDialog;
-import android.content.DialogInterface;
 import android.content.Intent;
-import android.os.AsyncTask;
 import android.os.Bundle;
 import android.util.Log;
-import android.view.LayoutInflater;
-import android.view.MenuItem;
-import android.view.View;
-import android.view.ViewGroup;
-import android.view.Window;
-import android.view.WindowManager;
-import android.widget.AdapterView;
-import android.widget.ArrayAdapter;
-import android.widget.BaseAdapter;
 import android.widget.Button;
 import android.widget.EditText;
-import android.widget.ImageView;
-import android.widget.PopupMenu;
-import android.widget.Spinner;
-import android.widget.TextView;
 import android.widget.Toast;
 
-import com.example.poste.PosteApplication;
 import com.example.poste.R;
-import com.example.poste.adapters.FolderAdapter;
-import com.example.poste.api.poste.API;
-import com.example.poste.api.poste.exceptions.APIException;
-import com.example.poste.api.poste.models.Folder;
-import com.example.poste.api.poste.models.FolderAccess;
-import com.example.poste.api.poste.models.User;
-import com.example.poste.api.poste.exceptions.IncompleteRequestException;
-import com.example.poste.api.poste.exceptions.MalformedResponseException;
-import com.example.poste.api.poste.models.Folder;
-import com.example.poste.api.poste.models.FolderAccess;
-import com.example.poste.api.poste.models.Post;
-
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.stream.Collectors;
+import com.example.poste.http.EditPostRequest;
+import com.example.poste.http.LoginRequest;
```

Comment:
Unused here I think

Reply:
Has since been optimized, good catch! leftover from copying code and auto-import.